src/styles/stars.css

```diff
@@ -0,0 +1,41 @@
+.card__block-stars {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: space-between;
+}
+
+.card__block-price {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: space-between;
+}
+
+.stars {
+  display: flex;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+  width: 96px;
+  height: 16px;
+}
+
+.stars-star {
+  height: 16px;
+  width: 16px;
+  background-image: url(/images/star-active.svg);
+  background-position: center;
+  background-repeat: no-repeat;
+  margin-right: 4px;
+}
+
+.stars--1 .stars-star:nth-child(n+5) {
+  background-image: url(/images/star.svg);
+}
+
+.card__text {
```

Comment:
Please reset default margin and move it to parent element
![image](https://user-images.githubusercontent.com/54936044/232194868-9e6c35f1-38b1-4a4a-82b9-9488b0f1cbcc.png)

![image](https://user-images.githubusercontent.com/54936044/232194806-dedc0282-45a8-48c8-95a7-ab1ccf80ef65.png)